src/clojure_repl_sufficient_p/defmulti.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-repl-sufficient-p.defmulti)
+
+(defmulti color-change (fn [x & _] (:animal x)))
+
+(defmethod color-change :octopus
+  [octopus color]
+  (assoc octopus :color color))
+
+(def pre-iteration-result (color-change {:animal :octopus :color :white} :red))
+
+(defmulti color-change (fn [x & _] (:or (:animal x)
+                                        (:plant  x))))
+
```

Comment:
For this to be a good case, we should have a `(defmethod color-change :hydrangea ...)` case at the minimum given the expectations of the test. 

As it turns out, I scrapped a multimethod namespace because it was surprising me with how well it behaved. Which is not to say we can't find edge cases, but we do need to be crisp about them actually being Clojure's problems. :)

Reply:
Agreed. It was an oversight.